op-e2e/actions/helpers/l2_proposer.go

```diff
@@ -117,7 +120,7 @@ func NewL2Proposer(t Testing, log log.Logger, cfg *ProposerCfg, l1 *ethclient.Cl
 
 	var l2OutputOracle *bindings.L2OutputOracleCaller
 	var disputeGameFactory *bindings.DisputeGameFactoryCaller
-	if e2eutils.UseFaultProofs() {
+	if cfg.AllocType == config.AllocTypeStandard {
```

Comment:
I suspect we shouldn't hard code standard as the only place fault proofs is used. In fact, we already have:
```suggestion
	if cfg.AllocType.SupportsProofs() {
```

Maybe `UsesProofs` would be a better name than `SupportsProofs` though.